Dockerfile

```diff
@@ -14,6 +14,11 @@ RUN bin/bash -l -c "gem install aws-sdk mime-types"
 RUN apt-get install -y python2.7
 ENV PYTHON /usr/bin/python2.7
 
+# install hugo
+ENV HUGO_VERSION 0.21
```

Comment:
We could also install hugo in `build.sh` instead of the container. That would give us a little more flexibility in the version. For example, we could do something like this to get the latest tagged release from the hugo repo:

```sh
echo "[build.sh] Installing hugo"

# Get the latest version tag from the GitHub releases
HUGO_VERSION=$(curl -sL -H 'Accept: application/json' https://github.com/spf13/hugo/releases/latest | jq -r '.tag_name')

# strip off the first character from the tag name, which is a 'v'
HUGO_VERSION="${HUGO_VERSION#?}"
echo "[build.sh] Latest version of hugo is ${HUGO_VERSION}"

curl -sSL https://github.com/spf13/hugo/releases/download/v${HUGO_VERSION}/hugo_${HUGO_VERSION}_macOS-64bit.tar.gz | tar zxf -
echo "[build.sh] $(./hugo version)"
```